File: beecrowd_exercicios_em_Ruby/begginer/keanu.rb
n = gets.to_i

n *= n

if n.even?
  puts "#{n / 2} casas brancas e #{n / 2} casas pretas"
else
  puts "#{n / 2 + 1} casas brancas e #{n / 2} casas pretas"
end

File: beecrowd_exercicios_em_Python/begginner/keanu.py
n = int(input())

n *= n

if n % 2 == 0:
    print("%d casas brancas e %d casas pretas" % (n // 2, n // 2))
else:
    print("%d casas brancas e %d casas pretas" % (n // 2 + 1, n // 2))
